addon/classes/Column.js

```diff
@@ -129,11 +130,37 @@ export default class Column extends EmberObject.extend({
 
   /**
    * The minimum width (in px) that this column can be resized to.
-   * @property minResizeWidth
+   * @property _minResizeWidth
    * @type {Number}
    * @default 0
+   * @private
+   */
+  _minResizeWidth: 0,
+
+  /**
+   * The minimum width that this column can be resized to.
+   * @property minResizeWidth
+   * @type {String}
+   * @default '0px'
    */
-  minResizeWidth: 0,
+  minResizeWidth: computed({
```

Comment:
Missing dependent key `_minResizeWidth`.

In practice this won't cause any problems yet, since `_minResizeWidth` is private API and exclusively managed by the `minResizeWidth` computed property. However, that could possibly change in the future and the developer making that change *then* might not remember to add that key.